en-US/mozorg/plugincheck-update.lang

```diff
@@ -0,0 +1,64 @@
+## NOTE: Demo server available at url: https://www-dev.allizom.org/plugincheck/
+## URL: https://www-dev.allizom.org/%LOCALE%/plugincheck
+
+
+# HTML page title
+;More browser features, fewer plugin updates | Firefox
+More browser features, fewer plugin updates | Firefox
+
+
+# HTML page decription
+# NPAPI stands for Netscape Plugin Application Programming Interface. It is an API that allows plugins (more specifically, browser extensions) to be developed for web browsers.
```

Comment:
Plugins and browser extensions are not the same thing. I would drop the second part of the doc